src/lib/viewers/controls/annotations/AnnotationsControls.tsx

```diff
@@ -5,10 +5,12 @@ import IconDrawing24 from '../icons/IconDrawing24';
 import IconHighlightText16 from '../icons/IconHighlightText16';
 import IconRegion24 from '../icons/IconRegion24';
 import useFullscreen from '../hooks/useFullscreen';
+import { AnnotationColor } from '../../../AnnotationModule';
 import { AnnotationMode } from './types';
 import './AnnotationsControls.scss';
 
 export type Props = {
+    annotationColor: AnnotationColor;
```

Comment:
Should this be an optional prop with a default?

Reply:
Yes I think it would be safer to make it optional + add a default just in case annotationColor is somehow undefined.